Image editor upload button has no aria-label #8215

skjulestad opened the pull request.

skjulestad committed:
Image editor upload button has no aria-label #8215

skjulestad committed:
Update ImageEditor.ts

alansemenov commented on modules/lib/src/main/resources/i18n/phrases.properties:
```diff
@@ -235,6 +235,7 @@ field.rich.text = Rich Text Editor
 field.rich.text.help = Write text and embed images
 field.fragment=Fragment
 field.fragment.help=Reusable components
+field.image.help = Upload or use existing image
```
You cannot select an existing image with the upload button.

alansemenov replied:
```suggestion
field.image.help = Upload an image file
```

alansemenov replied:
We also need to rename the key. The way it is now is too generic, and it doesn't have anything to do with field either.

We have a separate section of keys that start with "tooltip", so I suggest to add the new key there and call it "`tooltip.button.uploadImage`".

alansemenov replied:
```suggestion
tooltip.button.uploadImage = Upload an image file
```

skjulestad committed:
Update upload-image-button to be tab-able with label

Update upload image button using WCAG principles to make it tab-able and have the right label through tooltip.

alansemenov reviewed (approved):
Nice work, Mathias!
